### DISCUSSION TITLE
Appleton-Hartree Dispersion Relation

### DISCUSSION THREAD

**Discussion by KiRum89** [PlasmaPy/PlasmaPy#1547] [Dispersion]
I couldn't find the AH dispersion relation in the library. Has it been implmented yet? If not, I would like to add this feature to the "dispersion" package, if needed.

**Comment by rocco8773** [comment]
@KiRum89 That dispersion relation has not been implemented yet, so feel free take it on.  It would be much appreciated!  For examples to model you code after you can look to both the current functionality for the Bellan two fluid solution ([code](https://github.com/PlasmaPy/PlasmaPy/blob/main/plasmapy/dispersion/analytical/two_fluid_.py), [docs](https://docs.plasmapy.org/en/latest/api/plasmapy.dispersion.analytical.two_fluid_.two_fluid.html#plasmapy.dispersion.analytical.two_fluid_.two_fluid)) and the Hollweg solver ([code](https://github.com/PlasmaPy/PlasmaPy/blob/main/plasmapy/dispersion/numerical/hollweg_.py), [docs](https://docs.plasmapy.org/en/latest/api/plasmapy.dispersion.numerical.hollweg_.hollweg.html#plasmapy.dispersion.numerical.hollweg_.hollweg)).  We also have a Dispersion working group that meets every other week if you want to stop by and ask questions, see the [PlasmaPy calendar](https://calendar.google.com/calendar/embed?src=c_sqqq390s24jjfjp3q86pv41pi8%40group.calendar.google.com&ctz=America%2FNew_York).
